build.groovy

```diff
@@ -473,29 +473,15 @@ def finalizeBuildProcess(Map args) {
 
```

Comment:
Minor issue. Can you move the build report data file comment and variables down with the html file variables and where you actually create and save the file?